packages/framework/ai-collab/package.json

```diff
@@ -13,7 +13,7 @@
 	"sideEffects": false,
 	"type": "module",
 	"exports": {
-		".": {
+		"./alpha": {
```

Comment:
You'll need to point the `types` to `./lib/alpha.d.ts`, or this will end up including `@internal` APIs in addition to `@alpha` ones.

For consistency, it would be good to keep the `.` entry and have its `types` point to `./lib/public.d.ts`.

Reply:
We'll need to bring in some build patterns from other packages, right now this one doesn't generate different type definition files per entrypoint.

Reply:
I think we should do that in this PR then, to avoid potential issues.

Reply:
Updated the package.json in commit 31001d5dd2aeff5882330043fb792968cff702e1, we now generate index files as expected.